src/main/java/org/apache/maven/plugin/compiler/TestCompilerMojo.java

```diff
@@ -75,8 +75,11 @@ public class TestCompilerMojo extends AbstractCompilerMojo {
 
     /**
      * The directory where compiled test classes go.
+     * <p>
+     * This parameter should only be modified in special cases.
+     * See the {@code compile}-Mojo for more information.
```

Comment:
Render as:
![image](https://github.com/apache/maven-compiler-plugin/assets/3578921/d5114aad-a476-43b2-8f2c-0bf021a40925)
It was expected by you

maybe a link to compile-mojo documentation?

Reply:
Yes, that's exactly what I expected. But a link sounds reasonable, although I will want to avoid an import (ie will be qualified) and I doubt it will be used a lot...

Will update tomorrow